ircradio/templates/index.html

```diff
@@ -65,4 +85,9 @@ <h4>IRC</h4>
     </div>
   </div>
 </div>
+
+{% if ENABLE_SEARCH_ROUTE %}
+<script src="static/search.js"></script>
```

Comment:
`"static/search.js"` assumes the web-app is hosted on `/` 

generally you'd use `url_for()` to dynamically calculate the correct url and path